ts/features/bonus/bpd/components/superCashbackRanking/RankPositionItem.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { Badge, View, Text } from "native-base";
+import { StyleSheet } from "react-native";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import { H5 } from "../../../../../components/core/typography/H5";
+import { H4 } from "../../../../../components/core/typography/H4";
+import { formatNumberWithNoDigits } from "../../../../../utils/stringBuilder";
+import I18n from "../../../../../i18n";
+
+type Props = {
+  transactionsNumber: number;
+  superCashbackAmount: number;
+  boxedLabel: string;
+  rankingLabel: string;
+  currentUserPosition?: boolean;
+  hideBadge?: boolean;
+};
+
+const style = StyleSheet.create({
+  positionBox: {
+    paddingVertical: 8,
+    width: 48,
+    textAlign: "center"
+  },
+  badgeBlue: {
+    backgroundColor: IOColors.blue,
+    height: 18,
+    marginTop: 4
+  },
+  badgeText: {
+    fontSize: 12,
+    lineHeight: 18,
+    color: IOColors.white
+  }
+});
+
+const RankPositionItem: React.FunctionComponent<Props> = (props: Props) => (
```

Comment:
This is component is state-less so it's super easy to test?
Maybe you can write some UI tests in this story or in a dedicated one?

Reply:
I'm going to create and self assign a dedicated story to test this component and the entire Ranking component what do you think?